ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1025,6 +1020,34 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_pause_continue(const mavlink_comma
     return MAV_RESULT_DENIED;
 }
 
+MAV_RESULT GCS_MAVLINK_Copter::handle_command_nav_takeoff(const mavlink_command_int_t &packet)
+{
+    // takeoff altitude in centimeters above home
+    int32_t takeoff_alt;
```

Comment:
```suggestion
    // takeoff altitude in centimeters above home
    int32_t takeoff_alt_cm;
```

also, reduce the scope of this variable as much as possible (move it down to just above where it is used)